app/ledger.js

```diff
@@ -471,7 +471,12 @@ var updatePublisherInfo = () => {
   publisherInfo.synopsis = synopsisNormalizer()
 
   if (publisherInfo._internal.debugP) {
-    console.log('\nupdatePublisherInfo: ' + JSON.stringify(underscore.omit(publisherInfo, [ '_internal' ])))
+    data = []
+    publisherInfo.synopsis.forEach((entry) => {
+      data.push(underscore.extend(underscore.omit(entry, [ 'faviconURL' ]), { faviconURL: entry.faviconURL && '...' }))
```

Comment:
because the "real" string tends to be a long base64 value. all we really need to know is whether a value is present or not, the `"..."` serves that purpose... otherwise, you get a lot of scrolling in the console window with very little _useful_ content.